benchmarks/asv/app_suite.py

```diff
@@ -15,5 +15,17 @@
 Define asv benchmark suite that estimates the speed of applications.
 """
 
+from ..benchmark_functions.vqe import benchmark_vqe
 
 
+class VQEEvaluation:
+    """Benchmark VQE."""
+
+    params = ([1, 3], [False, True])
+    param_names = ['n_steps', 'optimize']
+
+    def time_hydrogen(self, n_steps, optimize):
+        """Time VQE for the hydrogen molecule."""
+        hyperparams = {'n_steps': n_steps,
+                       'optimize': optimize}
```

Comment:
Does optimize mean that we are using the grouping module? Has the bug been fixed?

Reply:
yep fixed it this morning